homeassistant/components/apprise/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "apprise",
+  "name": "Apprise",
+  "documentation": "https://www.home-assistant.io/components/apprise",
+  "requirements": [
+    "apprise==0.8.0"
+  ],
+  "dependencies": [],
+  "codeowners": []
```

Comment:
Thank you for your contribution thus far! 🎖 Since this is a new contribution, we would appreciate you'd added yourself to the list of code owners for this integration. ❤️ 

Please, add your GitHub username to the `manifest.json` of this integration.

For more information about "code owners", see: [Architecture Decision Record 0008: Code owners](https://github.com/home-assistant/architecture/blob/master/adr/0008-code-owners.md).

Reply:
Thank you; I've added my name to the `manifest.json` as requested.